### PR TITLE
feat(refreshratee): add refreshrate to the header

### PR DESCRIPTION
**Pull request by Poltergeist**
## Testing

Change you refresh rate setting to be not the default value of `2 seconds`. Ensure it is only showing if that applies.

## Screenshots

image

### CODE REVIEW

File: src/js/components/HeaderBar.tsx [hunk 1]
```diff
@@ -18,6 +21,19 @@ export default class HeaderBar extends React.Component {
           </Link>
         </div>
         <div className="header-bar-right-align-wrapper">
+          {refreshRate > 2 ? (
```
Comment (pierrebeitz) [inline src/js/components/HeaderBar.tsx]: if we want to show that if we're deviating from the default, why don't we express that here?

something like `currentRefreshRate != defaultRefreshRate`?

File: src/js/components/HeaderBar.tsx [hunk 2]
```diff
@@ -18,6 +21,19 @@ export default class HeaderBar extends React.Component {
...
+            <span>
+              <Tooltip
+                id="refreshRate"
+                trigger={`Refresh Rate: ${refreshRate} sec`}
+                maxWidth={150}
+              >
+                Data will only be requested every {refreshRate} seconds per
+                endpoint
```
Comment (pierrebeitz) [inline src/js/components/HeaderBar.tsx]: let's remove the "per endpoint" here as users most likely won't care and it also might not be true if we throttle repainting on some pages.

**Comment by pierrebeitz** [comment]
btw: what's the PR-title and commit message about?

**Comment by Poltergeist** [comment]
I messed up the commit I will create a new commit soon sorry :-( and therefore the title.

File: src/js/components/HeaderBar.tsx [hunk 3]
```diff
@@ -1,14 +1,17 @@
 import * as React from "react";
 import { Link } from "react-router";
 import { MountService } from "foundation-ui";
+import { Tooltip } from "@dcos/ui-kit";
 
 import ClusterHeader from "./ClusterHeader";
 import SidebarToggle from "./SidebarToggle";
+import Config from "../config/config";
```
Comment (pierrebeitz) [inline src/js/components/HeaderBar.tsx]: this path does not work on all unixes. your mac seems to handle paths case insensitive. you'll want to use `"../config/Config"`.

**Comment by pierrebeitz** [comment]
@Poltergeist added a fixup-commit (so please rebase before merging in case you like it).

**Comment by pierrebeitz** [comment]
ping @Poltergeist

**Comment by Poltergeist** [comment]
pong 🎾

**Pull request merged**